tuna/worker_interface.py

```diff
@@ -95,6 +98,15 @@ def __init__(self, **kwargs):
     #initialize tables
     self.set_db_tables()
 
+    self.job = types.SimpleNamespace()
+    try:
+      self.job_attr = [column.name for column in inspect(self.dbt.job_table).c]
+      self.job_attr.remove("insert_ts")
+      self.job_attr.remove("update_ts")
+    except NoInspectionAvailable:
+      #pass here for init_session
```

Comment:
I dont like silent failures, please add a LOGGER of some kind in case in the future this triggers for something other than init_session

Reply:
added logging